charts/openshift-metering/values.yaml

```diff
@@ -423,7 +426,11 @@ presto:
       config:
         tls:
           enabled: false
-          secretName: ""
+          serverSecretName: ""
+          rootCASecretName: ""
```

Comment:
Was there a reason this couldn't go into the same secret as the `serverCertificateSecretName`?